examples/Huggingface_Transformers/Transformer_handler_generalized.py

```diff
@@ -126,7 +129,7 @@ def preprocess(self, requests):
             max_length = self.setup_config["max_length"]
             logger.info("Received text: '%s'", input_text)
             # preprocessing text for sequence_classification and token_classification.
-            if self.setup_config["mode"] == "sequence_classification" or self.setup_config["mode"] == "token_classification":
+            if self.setup_config["mode"] == "sequence_classification" or self.setup_config["mode"] == "token_classification" or self.setup_config["mode"] == "text_generation":
```

Comment:
nit: let's break this into multiple lines

Reply:
maybe something like "if self.setup_config["mode"] in {"sequence_classification", "token_classification", "text_generation"}:" would be more elegant?